compose/container.py

```diff
@@ -131,10 +133,18 @@ def environment(self):
         return dict(var.split("=", 1) for var in self.get('Config.Env') or [])
 
     @property
+    def status(self):
+        return self.get('State.Status')
```

Comment:
well I am not using it directly inside docker-compose, I figured this could be helpfull for people like me who import compose directly into their python code. 

So I don't really mind if it does not make it through.